packages/core/echo/echo/src/Database.ts

```diff
@@ -137,12 +138,15 @@ export interface Database extends Queryable {
   // TODO(burdon): Can we move this into graph?
   get schemaRegistry(): SchemaRegistry.SchemaRegistry;
 
-  /**
-   *
-   */
   // TODO(burdon): Comment required.
   get graph(): Hypergraph.Hypergraph;
 
+  /**
+   * @deprecated Use `ref` instead.
+   */
+  // TODO(burdon): Rename getObject.
+  getObjectById<T extends Obj.Any = Obj.Obj<AnyProperties>>(id: string, opts?: GetObjectByIdOptions): T | undefined;
```

Comment:
why are we adding a deprecated method to this interface? why not just use `makeRef`?